torch/ao/ns/_numeric_suite_fx.py

```diff
@@ -189,12 +207,24 @@ def __init__(
         self.index_of_arg = index_of_arg
         # fully qualified name
         self.fqn = fqn
+        # if loggers are added before prepare_fx, but we do not want
+        # collect results of calibration, only results after convert_fx
+        # so, we add a flag to control whether this logger collects data
+        self.enabled = True
+        # string representation of qconfig
+        self.qconfig_str = qconfig_str
+        # this can be turned off to reduce memory usage during calibration
+        self.save_activations = True
 
     # Note: cannot annotate the type of x because TorchScript does not support
     #   the Union type.
     def forward(self, x):
         """
         """  # blank docblock to make autodoc happy
+        if not self.enabled:
+            return x
+        if not self.save_activations:
+            return x
```

Comment:
are these two flags doing the same thing?

Reply:
technically no, but I can see how the difference is not explained well.  `self.enabled` is controlling if the logger is doing anything at all, and `save_activations` is only controlling whether the logger is saving activations.  A logger could be enabled and be saving comparisons, but not activations. I'll clean this up.